build.gradle.kts

```
@@ -32,7 +32,7 @@ allOpen {
 }
 
 group = "com.ampnet"
-version = "0.15.0"
+version = "0.15.1"
```

```suggestion
version = "0.16.0"
```
It's a breaking change if it brakes proto, see `enum Role`